openslides_backend/action/actions/mediafile/upload.py

```diff
@@ -87,13 +88,23 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         instance["create_timestamp"] = round(time())
         filename_ = instance.get("filename", "")
         file_ = instance.pop("file")
-        instance["mimetype"] = mimetypes.guess_type(filename_)[0]
-        if instance["mimetype"] is None:
-            raise ActionException(f"Cannot guess mimetype for {filename_}.")
         decoded_file = base64.b64decode(file_)
+        mimetype_ = magic.from_buffer(decoded_file, mime=True)
+        possible_extensions = mimetypes.guess_all_extensions(mimetype_)
```

Comment:
You could chance the sequence of line 92 and 93. No need to guess the possible_extensions of mimetype_ None

Reply:
Done